changelog.md

```diff
@@ -0,0 +1,86 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+This project adheres to [Semantic Versioning](http://semver.org/).
+
+## [Unreleased](https://github.com/silverstripe/silverstripe-widgets/tree/HEAD)
```

Comment:
We tend not to include unreleased changes in change log - would you mind removing this section?